### PR TITLE
Add LDAPVerifyServerCert directive

### PR DESCRIPTION
**Pull request by deecay**
This PR adds control over LDAPVerifyServerCert.

### CODE REVIEW

File: proxy_ldap.conf.template [hunk 1]
```diff
@@ -1,3 +1,7 @@
+$([[ -v NO_VERIFY_SERVER_CERT ]] && { echo "
```
Comment (dariko) [inline proxy_ldap.conf.template]: I think this block should be placed after the `LoadModule ldap_module` line , possibly just before the
```
$([[ -v LDAPS_CACERT_PEM ]] && { echo "
```

**Review by dariko — CHANGES_REQUESTED** [review]
Looks good. I left an inline comment that should be worked out before merging.
Thanks

File: proxy_ldap.conf.template (hunk 1 above)
Comment (deecay) [reply proxy_ldap.conf.template]: Changed.

**Review by dariko — APPROVED** [review]

**Comment by dariko** [comment]
thank you!

**Pull request merged**